packages/material/src/mui-controls/MuiInputText.tsx

```diff
@@ -51,18 +67,22 @@ export class MuiInputText extends React.Component<
       isValid,
       path,
       handleChange,
-      schema
+      schema,
+      inputProps
```

Comment:
Maybe we could rename `inputProps` to `muiInputProps` and keep `inputProps` in the body?